tools/CouchDB_Confirm_Integrity.php

```diff
@@ -18,7 +18,7 @@
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
 
-
+require_once __DIR__ . "/../vendor/autoload.php";
```

Comment:
I think generic_includes already does this, but it shouldn't harm anything to require_once twice.